lib/kino/control.ex

```diff
@@ -81,6 +88,27 @@ defmodule Kino.Control do
   This widget is represented as button that toggles interception
   mode, in which the given keyboard events are captured.
 
+  > #### Keyboard shortcut {:.info}
+  >
+  > As of Livebook v0.10.1, keyboard controls can be toggled by
+  > focusing the cell and pressing `Ctrl + k` (or `Cmd + k` on
+  > MacOS).
+
+  ## Options
+
+  Note that these options require Livebook v0.10.1 or later.
```

Comment:
Need to confirm that the version here and in the note above is actually the next planned Livebook release version.

Reply:
```suggestion
  Note that these options require Livebook v0.11 or later.
```